sympy/polys/tests/test_rings.py

```diff
@@ -1363,6 +1363,28 @@ def test_PolyElement_pdiv():
     assert f.pquo(g) == q
     assert f.pexquo(g) == q
 
+
+def test_PolyElement_sparse():
+
+    _, x, y, z = ring("x, y, z", ZZ)
+    p1 = 4*x**3 + 5*y**2 + 6*z + 7
+    dg, sym = 2, 1
+    assert p1.coeff_wrt(sym, dg) == 5
```

Comment:
This example does not really demonstrate the purpose of the `coeff_wrt` method because it only gathers a single coefficient.

Reply:
This was marked as resolved but it does not seem to be resolved to me. The test here does not really demonstrate what this method does or what its purpose is because the same output would be obtained from the `coeff` method:
```python
In [26]:     _, x, y, z = ring("x, y, z", ZZ)
    ...:     p1 = 4*x**3 + 5*y**2 + 6*z + 7

In [27]: p1.coeff(y**2)
Out[27]: mpz(5)
```
It is fine to have this test but there also need to more tests that clearly show what this method does. If someone changes the `coeff_wrt` method in future then the tests should show if it is working correctly.